### PR TITLE
Moving contributors/devel/help-wanted.md to contributors/guide-  Issue 3064 3

### PR DESCRIPTION
**Pull request by eduartua**
Related to #3064 

Moving help-wanted.md from /devel to contributors/guide.

/sig contributor-experience
/kind feature
/area contributor-guide

/cc @parispittman @nikhita 

/assign @guineveresaenger

### CODE REVIEW

**Comment by guineveresaenger** [comment]
/ok-to-test

Before I LGTM:
- could you squash your commits?
- could you file an issue/PR in k/test-infra to change the link here:https://github.com/kubernetes/test-infra/blob/master/prow/plugins/help/help.go ? Because you're putting in a stub, this is _probably_ not going to break things, but we do want to follow up

**Comment by eduartua** [comment]
Done. commits squashed.
@guineveresaenger

**Comment by guineveresaenger** [comment]
/lgtm

This is great!

File: contributors/guide/help-wanted.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+questions, that people will let them know the rules/norms, that their
+contributions are helpful and appreciated... they will stick around! 🌈
+- Encourage new contributors to seek help on the appropriate slack channels,
+  introduce them, and include them in your conversations.
+- Invite them to the SIG meetings.
+- Give credit to new contributors so that others get to know them, _"Hey, would
+  someone help give a second LGTM on @newperson's first PR on chocolate
+  bunnies?"_. Mention them in the SIG channel/meeting, thank them on twitter or
+  #shoutouts.
+- Use all the emoji in your approve or lgtm comment. 💖 🚀
```
Comment (krmayankk) [inline contributors/guide/help-wanted.md]: lol

**Pull request merged**